### PR TITLE
Orchestrator with multiple tests 

### PR DESCRIPTION
**Pull request by RohanAhluwalia**
New merge for orchestrator into the paper benchmarking branch. Should be able to merge into the paper benchmarking branch as of 7/20. 

To run tests for orchestrator run, `cargo test --features="full-ci, channel-async-std" -p hotshot-testing --test orchestrator -- --nocapture`

### CODE REVIEW

File: orchestrator/src/lib.rs [hunk 1]
```diff
@@ -194,12 +248,8 @@ where
     KEY: serde::Serialize,
     ELECTION: serde::Serialize,
 {
-    let api_toml = toml::from_str::<toml::Value>(include_str!(concat!(
```
Comment (nomaxg) [inline orchestrator/src/lib.rs:197]: let's leave the instantiation of the API as it is, if we create the API from a file, external crates wont be able to run the orchestrator

Comment (RohanAhluwalia) [reply orchestrator/src/lib.rs:197]: Yes! Makes sense. Updated it

**Review by elliedavidson — COMMENTED** [review]
Looks good!  The tests pass!  I'd like to wait to merge until the sequencer team is stable on using this branch for their code.

**Comment by rob-maron** [comment]
Closing for now but keeping the branch open !

**Pull request closed without merging**